packages/common/src/hooks/useFilterBy/useFilterBy.tsx

```diff
@@ -1,6 +1,6 @@
 import { useState } from 'react';
 
-interface FilterByConditionByType {
+export interface FilterByConditionByType {
```

Comment:
only the types will be used from the existing hooks, so will just lift those out